src/main/java/com/smartcar/sdk/data/BatchResponse.java

```diff
@@ -0,0 +1,105 @@
+package com.smartcar.sdk.data;
+
+import com.smartcar.sdk.SmartcarException;
+
+import org.joda.time.DateTime;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.GsonBuilder;
+import org.apache.commons.text.CaseUtils;
+import java.util.HashMap;
+import java.util.Iterator;
+
+public class BatchResponse extends ApiData {
+  private HashMap<String, JsonObject> responseData = new HashMap<>();
+
+  public BatchResponse(JsonArray responses) {
+
+    Iterator<JsonElement> responseIterator = responses.iterator();
+
+    while (responseIterator.hasNext()){
```

Comment:
Nit consistency on spacing.
```suggestion
    while (responseIterator.hasNext()) {
```